src/osm_configurator/view/utilityframes/reduction_default_value_frame.py

```diff
@@ -530,12 +534,11 @@ def freeze(self):
 
             self._tag_list.freeze()
 
-
             self._frozen: bool = True
 
     def unfreeze(self):
         """
-        If this method is called, the frame returns into its previous interactable state.
+        If this method is called, the frame returns into its previous intractable state.
```

Comment:
the is nothign that will be intracted, the is only interacting or no interacting! No intracting!

Reply:
ok